### ISSUE TITLE
Dogecoin on Kubernetes! (Ongoing effort already underway)

### ISSUE THREAD

**Issue by sbrooks-aither** [dogecoin/dogecoin#2195]
**Preface**: While trying to get started to see how I can contribute to Dogecoin, I found a lack of very clear guidance on best practices for running DogecoinD, especially in a server environment. Standardized and repeatable deployments with accompanying Documentation and Guides for operations would go a very long way to lower the barrier of entry into this community. Therefore, i'm making some things. I am posting this announcement here because:

1) This was posted to r/dogecoin but got drowned out by memes
2) I attempted to post to r/dogecoindev but Karma restrictions block any new users from contributing meaningfully or even posting
3) Tried to message the sub moderators but only got radio silence. I'm sure they're busy writing code instead of policing the internet

-------------------------------------------------

   At Aither Architects, our mission is to be an enabling factor and a force multiplier for Makers, Doers, and anyone with the creative spirit -- as _**that**_ is what drives humanity forward. Whether you’re looking to showcase your creations, start **_and continuously operate_** your business, or organize a movement for the positive benefit of **_all humans_**, we’ll be right by your side to help provide the support and technical foundations for you to be you.

   Aither’s values and morals line up with what Dogecoin’s potential and its community represent, so we will proudly commit to its adoption and growth. Even though Aither is a pre-launch startup, that doesn’t mean we’re unable to give to the community and I want our first public act to be one of giving and a gesture of goodwill. Dogecoin has massive fervor behind it; however, user adoption is a problem. That’s why today i’m proud to announce the alpha release and ongoing development of a Helm Chart and Docker images for running a Dogecoin FullNode on Kubernetes. It's rough and early, but I felt it more important to get this out to the world for collaboration opportunities instead of trying to deliver something overly polished 2 months from now. This will help standardization and make it easier to run Dogecoin in a server environment, develop for it, mine coins, and help secure the network. Most importantly, it’s a step in the right direction for reducing the barrier of entry into crypto. Using this vehicle, we can:

- Make it easier for anyone to run and operate Dogecoin FullNodes
- Enhance developer experience and tap into the massive Kubernetes ecosystem
- Lower the barrier of entry into crypto for education and for make benefit glorious community projects of Doge!
- Use it as a starting point for a Community Dogecoin DevOps program

https://github.com/Aither-Architect/dogecoin-docker

https://github.com/Aither-Architect/dogecoin-helm

https://hub.docker.com/r/aitherarchitect/dogecoin/tags

   It's one thing to say we're going to Do Only Good Everyday, but it's another thing altogether to say exactly what that looks like. That’s an immensely complex topic owing to the fact that ‘good’ looks different to each of us. The most important thing of all that we need to keep in mind, which eclipses everything else and fuels this just cause, is our Why. Like so many of you, this coin --for me-- isn’t about novelty or beating another cryptocurrency, but a chance at some life altering income that represents security for my family and loved ones as well as setting up a fair and transparent financial system for humanity. It all boils down to hope in a brighter future, which can move mountains. In a world seemingly designed to squeeze us for all we’re worth, we could all use a little hope… and i’m willing to fight for that. What’s your DogeWhy?

As always, PULL REQUESTS ARE WELCOME!!!!!

Sean Brooks
Chief Architect & CVO

**Comment by patricklodder** [comment]
@sbrooks-aither we have been working on the [dogecoin/docker](https://github.com/dogecoin/docker) repository to make some progress on standardizing a base docker image. It currently supports x86_64, i686, armv7 and aarch64 binaries.

With your permission, I'd like to transfer this issue there, so that we can collaborate on this further. Let me know what you think please.

**Comment by sbrooks-aither** [comment]
@patricklodder Yes, of course; whatever is necessary for the community. Please forgive my radio silence for so long. A whole lot of life hit me these past 6 months. I'm back in the saddle, though, and should be producing some content and PRs in the relatively near future.
